App.js

```diff
@@ -63,8 +68,8 @@ class App extends Component {
         transforms: [encryptor],
         whitelist: ['auth'],
       },
-      () => {
-        this.setState({ rehydrated: true });
+      (err, { auth }) => {
+        this.setState({ rehydrated: true, language: auth.locale });
```

Comment:
This fails when we're not authenticated and loading the app from scratch:

![simulator screen shot - iphone 6 - 2018-03-30 at 17 35 31](https://user-images.githubusercontent.com/12476932/38155446-ddfd1484-3445-11e8-8df9-a63d655af082.png)

Maybe we can default the locale if the store is empty like `this.setState({ rehydrated: true, language: auth.locale || 'en' });`?

Reply:
Definitely :D